tutorial_3_django_models/slides.md

```diff
@@ -0,0 +1,207 @@
+    - *python manage.py migrate*
+  - Output:
+    - *Show command line output for makemigrations and migrate*
+    - Observe that a *db.sqlite3* has been created.
+    - Check the /blog/migrations/ directory and you will observe that a new file has been created.
+  - We will be revisiting *migrations* in more detail later
+
+Slide 5: What is Django Admin App [00:07| 06:31]
+-------------------------------------------
+
+  - What is the Django Admin App
+    - The django admin app is a feature that provides an interface to Create, Read, Update and Delete model instances
+
+Demonstration: Create a superuser [00:55| 07:26]
+----------------------------------------------
+  - Create a superuser to login to the admin interface
+    - Run *python manage.py createsuperuser*
+      - Provide a username
+      - Provide an email address
+      - Provide a password and confirm it by entering again
+  - Output: *Superuser created successfully.*
+
+
+Demonstration: Show the Admin interface [00:45| 08:11]
+---------------------------------
+  - Run the django server with
+    - *python manage.py runserver*
+  - Open the web browser and go to http://127.0.0.1:8000/admin/
+  - You will see the login screen
+      - [Add Screenshot]
+  - Enter the superuser credentials as created previously
+  - You can see the interface
+    - [Add Screenshot]
+    
+Demonstration: Make the Blog App modifiable through Admin [01:10 | 09:21]
+-------------------------------------------
+  - In order to display the Blog app and it's components in the admin interface, change the */blog/admin.py* file;
+
+    # /blog/admin.py
+    from django.contrib import admin
+
+    from .models import Blog, Article
+
+    admin.site.register(Blog)
+    admin.site.register(Article)
+  - Save
+  - Login to the Admin interface
+
+Demonstration: Add a Blog database entry [00:45| 10:06]
+-----------------------
+  - Click on 'Blog'
+  - Click on 'Add Blog'
+  - Fill the form for new blog
+    - name
+    - creator
+    - create_date
+  - Click on Save
+  
+  - You will see your new blog in the list
+    - You can click on the blog name to view it's details
+  - Click on Home (in breadcrumb section)
+  
+Demonstration: Add an Article database entry [00:50| 10:56]
+--------------------------------
+  - Click on 'Article'
+  - Click on 'Add Article'
+  - Fill the form for the new Article
+    - blog: Select 'My New Blog' from list
+    - title
+    - create_date
+    - author: Select super-username form the list
+    - body: Add Article text
+  - Click on Save
+  - You will see your new article in the list
+    - You can click on the article name to view it's details
+    
+
+So this is a web application framework.
+Slide 6 [00:17 | 11:13]
+----------------
+
+**What is a web application**
+
+![Block diagram of Web application](https://raw.githubusercontent.com/FOSSEE/learn_django/master/tutorial_1_intro/webapp_diag.png 'Web Application Block diagram')
+** Can be used for narration for this slide **	
```

Comment:
**Can be used for narration for this slide** (remove the white space between asterisk)